MAUI/PDF-Viewer/Printing-PDF-files.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: Printing PDF files in .NET MAUI PDF Viewer control | Syncfusion
+description: Learn here all about printing feature in Syncfusion .NET MAUI PDF Viewer (SfPdfViewer) control and more.
+platform: MAUI
+control: SfPdfViewer
+documentation: ug
+---
+
+# Printing PDF Files in .NET MAUI PDF Viewer (SfPdfViewer)
+
+![Printing PDF Files in .NET MAUI PDF Viewer](Images\Print\print.png)
+
+![Print Preview Window](Images\Print\print-preview-window.png)
+
+
+Using [SfPdfViewer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html), you can effortlessly print PDF documents directly from your application code. This feature enables users to conveniently fill PDF form fields and print the document with the updated values incorporated.
+
+Moreover, when performing a print operation, the default system print dialog opens, providing users with familiar options and settings to customize their printing experience. Within this dialog, users can adjust settings such as printer selection, the number of copies, page orientation, and more, ensuring that their printed documents meet their specific requirements.
+
+To print a PDF programmatically, you can use the `PrintDocument()` method provided by SfPdfViewer. This method enables you to trigger the printing action seamlessly within your application.
+
+{% tabs %}
+{% highlight c# %}
+// Prints the PDF document.
+PdfViewer. PrintDocument();
+{% endhighlight %}
+{% endtabs %}
+
+Also, SfPdfViewer provides another way to print using the `PrintDocumentCommand`. This makes printing easier, especially when used with buttons.
+
+{% tabs %}
+{% highlight xaml %}
+<!-- Prints the PDF document. -->
+<Button Text="&#xe77f;" Command="{Binding Source={x:Reference PdfViewer},Path=PrintDocumentCommand}"/>
+{% endhighlight %} 
+{% endtabs %}
+
+## Limitations
+
```

Comment:
<img width="346" alt="image" src="https://github.com/syncfusion-content/maui-docs/assets/65169077/9e7bdc37-cb11-4a91-915f-1f7fab096e7a">

1. Check the grammatical.
2. Missing some key words like 
"default" - Comment is default 
"always" - always irrespective of its appearance.
"Currently" - To denote it is temporary

Reply:
Changed .